### PR TITLE
Create the application template.

### PR DESCRIPTION
**Pull request by rahmot**
Hi! I just created the application templates for both mentors and mentees. I will update the `readme` later. Pushed this so that I can switch to `about` branch to work on it. Kindly let me know if there is anything to fix. Thanks!

### CODE REVIEW

File: application_template/Mentor_application.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Our Mentee Application Template
+
+Hello! Thank you for your interest in being a part of Galaxy Mentoring Network!
+
+**Kindly download this template so that you can have a draft before you tranfer your responses to the form.**
```
Comment (bebatut) [inline application_template/Mentor_application.md:5]: Download will likely not working there. Maybe better to say "Copy / paste"

**Review by bebatut — COMMENTED** [review]
Could you rename the mentor application file to use only lower case?

Where will people find the link to the templates? Should it be in the navbar? If yes, could you add the links in the navbar in this PR?

File: application_template/Mentor_application.md (hunk 1 above)
Comment (rahmot) [reply application_template/Mentor_application.md:5]: Thinking out loud, should we embed the templates (doc) to the application guideline page?

**Comment by bebatut** [comment]
Can you put the files in `src/pages`?

File: content/application.md [hunk 2]
```diff
@@ -40,8 +40,9 @@ As soon as a match occurs, the network manager will send mails to the mentee(s)
 
 ## How to apply
 
-1. Click on the application link to fill the form.
-2. Answer all questions (Tip: be yourself, we want to know you and what you want to achieve with this mentorship).
-3. Review your answers and click submit.
+1.  Check the (application template)[#].
```
Comment (bebatut) [inline content/application.md:43]: Can you update with the link?

**Comment by bebatut** [comment]
Sorry I messed up with the location. Can you put all files in `content` and add corresponding Vue files in `src/pages`?

File: content/application.md (hunk 2 above)
Comment (rahmot) [reply content/application.md:43]: I can't do that now because the application template PR hasn't been merged. I will do that later.

Comment (bebatut) [reply content/application.md:43]: I am not sure to understand why you can not add the link. The files are in this PR. Once you created the Vue files you will be able to check the pages and add the links there

File: content/application.md [hunk 3]
```diff
@@ -40,8 +40,11 @@ As soon as a match occurs, the network manager will send mails to the mentee(s)
 
 ## How to apply
 
-1. Click on the application link to fill the form.
-2. Answer all questions (Tip: be yourself, we want to know you and what you want to achieve with this mentorship).
-3. Review your answers and click submit.
+1.  Check the application template:
+    - (Mentee Application Template)[/content/mentee_application/]
```
Comment (bebatut) [inline content/application.md:44]: Without `/content`

**Review by bebatut — COMMENTED** [review]
Did you build the website locally? Could you see how the pages look like?

File: content/application.md (hunk 3 above)
Comment (rahmot) [reply content/application.md:44]: Yes, it shows localhost:8080/mentee_application. Is that right?

**Review by bebatut — COMMENTED** [review]
Could you use the same name for the Vue pages as for the Markdown file? Thanks

File: content/mentee_application.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+# Our Mentee Application Template
```
Comment (bebatut) [inline content/mentee_application.md:1]: Can you add the metadata on the top of both Markdown files as the other Markdown files? Thanks

**Review by bebatut — COMMENTED** [review]
The building on netlify is failing.
I looked at the logs: `Could not generate HTML for "/mentee-application/"

I checked the [Gridsome doc](https://gridsome.org/docs/pages/):
- The Vue page should be named `MenteeApplication,vue` and `MentorApplication.vue`
  -  the generated pages will be `mentee-application` and `mentor-application`
- The Markdown should be named `mentee-application.md`  and `mentor-application.md`

Could you rename the files this way and also the links?

File: content/mentee-application.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+6. Pronouns
+7. Affiliations
+8. Age (in ranges:) ***Choose one option***
+    20-30
+    30-40
+    40-50
+    50 - 60
+    60-70
+    70+
+    Prefer not to say
```
Comment (bebatut) [inline content/mentee-application.md:30]: Could you put as a list?

File: content/mentee-application.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+This is to know a bit about how about you and how best we can support during the mentoring program.
+
+9.  Educational Level ***Choose one option***
+    High School
+    Undergraduate
+    Postprogarduate
+    Postdoctoral
+    Engineer
+    PI
+    Other
```
Comment (bebatut) [inline content/mentee-application.md:43]: Can you put as a list?

File: content/mentee-application.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+10. Current Occupation ***Choose one option***
+    Employed
+    Self-employed
+    Freelancing
+    Student
```
Comment (bebatut) [inline content/mentee-application.md:48]: List

Comment (bebatut) [reply content/mentee-application.md:48]: And same for all other similar questions

File: content/mentee_application.md (hunk 4 above)
Comment (rahmot) [reply content/mentee_application.md]: > The building on netlify is failing. I looked at the logs: `Could not generate HTML for "/mentee-application/"
> 
> I checked the [Gridsome doc](https://gridsome.org/docs/pages/):
> 
> * The Vue page should be named `MenteeApplication,vue` and `MentorApplication.vue`
>   
>   * the generated pages will be `mentee-application` and `mentor-application`
> * The Markdown should be named `mentee-application.md`  and `mentor-application.md`
> 
> Could you rename the files this way and also the links?

Thanks so much for looking into that. It is perfecly working now. :)

File: content/mentee-application.md (hunk 5 above)
Comment (bebatut) [reply content/mentee-application.md:30]: It looks strange: 
image

File: content/mentee-application.md (hunk 7 above)
Comment (rahmot) [reply content/mentee-application.md]: Done, thanks for pointing!

**Comment by bebatut** [comment]
Thanks!

**Pull request merged**